src/cargo/ops/cargo_doc.rs

```diff
@@ -16,6 +18,13 @@ pub struct DocOptions {
     pub compile_opts: ops::CompileOptions,
 }
 
+#[derive(Deserialize)]
+struct CargoDocConfig {
+    /// Browser to use to open docs. If this is unset, the value of the environment variable
+    /// `BROWSER` will be used.
+    browser: Option<String>,
```

Comment:
I'm thinking this should maybe be `PathAndArgs`, so that there is some consistency with other config options that specify executables to run.